src/SortByNameDown.java

```diff
@@ -0,0 +1,8 @@
+import java.util.Comparator;
+
+public class SortByNameDown implements Comparator<Player> {
```

Comment:
SortPlayerNameDescending would be a better name. Ascending and Descending are keywords in sorting instead of Up and Down.